packages/common/src/controllers/file-upload.controller.ts

```diff
@@ -54,6 +54,10 @@ export class FileUploadController {
     @Res() res: FastifyReply,
   ): Promise<void> {
     try {
+      const pathTraversalRegex = /\.\.\//;
```

Comment:
will it handle all cases? i think the regex should just be allowing the `<A-Z|0-8>.<A-Z>` with the extension being optional. Please add relevant test cases to show that it works.